Update to 0.12.1, use abi3 builds

effigies opened the pull request:
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #42.

effigies committed:
Use abi3

regro-cf-autotick-bot committed:
updated v0.12.1

effigies commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you but ran into some issues. Please check the [output logs](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19167671083) of the GitHub Actions workflow below for more details. You can also ping `conda-forge/core` (using the `@` notation) for further assistance or you can try [rerendering locally](https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-with-conda-smithy-locally).


The following suggestions might help debug any issues:
* Is the `recipe/{meta.yaml,recipe.yaml}` file valid?
* If there is a `recipe/conda-build-config.yaml` file in the feedstock make sure that it is compatible with the current [global pinnnings](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml).
* Is the fork used for this PR on an organization or user GitHub account? Automated rerendering via the webservices admin bot only works for user GitHub accounts.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19167671083](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19167671083). Examine the logs at this URL for more detail.</sub>

effigies committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.11.06.19.20.54

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.9

effigies commented:
@conda-forge/help-osx-arm64 I'm trying to convert this Python package to use abi3 builds for 3.11+, and this is working for all but osx-arm64. I don't really understand the build process enough to fix this based on the CI logs, and I don't have a Mac to build locally.

isuruf committed:
python_min in build.

effigies commented:
Thanks @isuruf!
